src/styles/helper.css

```diff
@@ -151,15 +151,9 @@ textarea {
   transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
 }
 
-/** Spacing */
-.min-w-100 {
+* Spacing */ .min-w-100 {
```

Comment:
This is an invalid css comment. Everything after this line will not be processed.

Reply:
Resolved